src/sinks/blackhole.rs

```diff
@@ -80,23 +81,19 @@ impl BlackholeSink {
 
 #[async_trait]
 impl StreamSink for BlackholeSink {
-    async fn run(&mut self, mut input: BoxStream<'_, Event>) -> Result<(), ()> {
-        while let Some(event) = input.next().await {
+    async fn run(&mut self, input: BoxStream<'_, Event>) -> Result<(), ()> {
+        let mut chunks = input.chunks(128);
```

Comment:
Would it make sense to maybe tie this to `print_amount`? `128` seems a bit arbitrary (we may want to leave a comment if we leave it as-is about why this value was chosen).

Reply:
It's just an arbitrary low number. Pegging it to `print_amount` makes sense to me.

Reply:
I think you also need to update the acker at the bottom. It is still acking one event per loop.